physics/radiation_clouds.f

```diff
@@ -2216,8 +2217,12 @@ subroutine progcld_thompson_wsm6                                  &
 
         do k = 1, NLAY-1
           do i = 1, IX
-            cwp(i,k) = max(0.0, clw(i,k,ntcw) * gfac * delp(i,k))
-            cip(i,k) = max(0.0, clw(i,k,ntiw) * gfac * delp(i,k))
+            !cwp(i,k) = max(0.0, clw(i,k,ntcw) * gfac * delp(i,k))
```

Comment:
Is it correct that the same convective condensate is added to both the liquid and ice water paths?